src/main/java/hw3/utils/Utils.java

```diff
@@ -0,0 +1,35 @@
+package hw3.utils;
+
+import org.openqa.selenium.WebElement;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Utils {
+    public static List<String> getTextFromWebElementsInUpperCase(List<WebElement> webElements) {
+        List<String> textList = new ArrayList<>();
+        for (WebElement element : webElements) {
+            textList.add(element.getText().toUpperCase());
+        }
+        return textList;
+    }
+
+    public static String toNormalCase(String input) {
+        if (input == null || input.isEmpty()) {
+            return input;
+        }
+        StringBuilder converted = new StringBuilder();
+        boolean convertFirst = true;
+        for (char ch : input.toCharArray()) {
+            if (convertFirst) {
+                ch = Character.toTitleCase(ch);
+                convertFirst = false;
+            } else {
+                ch = Character.toLowerCase(ch);
+            }
+
+            converted.append(ch);
+        }
+        return converted.toString();
+    }
```

Comment:
Why do you need this method?

Reply:
I use this method to find elements on page. Searching element is a text parameter; on page this text could be in upper case or in normal case. So to be sure the parameter is in correct case, I apply this method.
For exemple, hw3.ex2.ServicePageSteps#goTo accept ENUM MenuElements as a parameter. Enum values are all in the upper case. But to find them on header menu, normal case is needed (xpath is used).